### PR TITLE
CN-4506 celpay info masked

### PR DESCRIPTION
**Pull request by ShamanShaulic**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/utils/transactions-util.js [hunk 1]
```diff
@@ -15,14 +16,77 @@ const transactionsUtil = {
  * @returns {Object}
  */
 function mapTransaction(transaction) {
-  const newTransaction = { ...transaction };
+  let newTransaction = { ...transaction };
+
+  newTransaction = maskCelPayUser(newTransaction);
```
Comment (fjovakaric) [inline app/utils/transactions-util.js]: move one line below, under getTransactionType

File: app/utils/transactions-util.js [hunk 2]
```diff
@@ -15,14 +16,77 @@ const transactionsUtil = {
...
+ * Masks senders and receivers first name, last name and their emails
+ *
+ * @param {Object} transaction
+ * @returns {Object}
+ */
+function maskCelPayUser(transaction) {
+  const newTransaction = { ...transaction };
+
+  if (
+    newTransaction.coin === "cel" &&
```
Comment (fjovakaric) [inline app/utils/transactions-util.js]: should be all coins
use newTransaction.type.includes("CELPAY")

File: app/utils/transactions-util.js [hunk 3]
```diff
@@ -15,14 +16,77 @@ const transactionsUtil = {
...
+      );
+    }
+
+    if (newTransaction.transfer_data.sender.email) {
+      newTransaction.transfer_data.sender.email = formatter.maskEmail(
+        newTransaction.transfer_data.sender.email
+      );
+    }
+
+    return newTransaction;
```
Comment (fjovakaric) [inline app/utils/transactions-util.js]: remove line
